index.html

```diff
@@ -11,5 +11,38 @@
     <title>My Portfolio2</title>
 </head>
 <body>
+  <div class="gen-container">
+    <nav>
+      <h3 class="logo">
+        My Logo
+      </h3>
+      <div class="menu-bars">
+        <img src="header-section\bars.svg" alt="">
+      </div>
+    </nav>
```

Comment:
Kindly wrap this block in a `header` tag. Also, move the logo outside the `nav` tag into the `header` 🤗

- Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.